collector/agentPoller.go

```diff
@@ -429,8 +406,7 @@ func (a *Agent) getAgentState(stats chan<- statsd.StatsEvent) (*AgentState, erro
 		executorAppNames: executorAppNames}, nil
 }
 
-func (a *Agent) getJSONFromAgent(urlPath string, testFileFlag *string, stats chan<- statsd.StatsEvent) (*jason.Value, error) {
-	stats <- statsd.MakeEvent(statsd.AgentQuery)
+func (a *Agent) getJSONFromAgent(urlPath string, testFileFlag *string) (*jason.Value, error) {
 	var rawJSON []byte
```

Comment:
Agreed, already done but haven't committed / pushed yet. It's ticketed internally though.